### DISCUSSION TITLE
Did not find X_scvi in `.obsm.keys()`.

### DISCUSSION THREAD

**Discussion by dxl3742** [tuonglab/dandelion#303] [Q&A]
Hi Kelvin, 
Great work on the package!
I want to run the trajectory analysis on my seurat object, which I have converted and preprocessed according to your instructions. 
however I have run into this bug:
>>> sc.pp.neighbors(adata, use_rep="X_scvi", n_neighbors=50)
ValueError: Did not find X_scvi in `.obsm.keys()`. You need to compute it first.
>>> print(adata.obsm.keys())
KeysView(AxisArrays with keys: X_harmony, X_pca, X_umap)

How do I fix this? Any help would be appreciated, thanks in advance!

**Comment by zktuong** [comment]
Hi @dxl3742,

Thanks for the interest! Yea should swap that out with the keys you have, which should be either `X_harmony` or `X_pca`.

For example:
```python
sc.pp.neighbors(adata, use_rep="X_harmony", n_neighbors=50)
```

Having said, that, if you already have a neighborhood graph in your `.obsm` (that you used for your normal analysis), you can skip this step and continue with the remaining steps.
